src/lib/components/event/event-summary-row.svelte

```diff
@@ -250,6 +250,11 @@
   onMount(async () => {
     if (isLocalActivityMarkerEvent(event)) {
       primaryLocalAttribute = await decodeLocalActivity(event);
+    } else if (
+      isEventGroup(event) &&
+      isLocalActivityMarkerEvent(event.initialEvent)
+    ) {
```

Comment:
DUDE! I should have thought of this! Thanks for the update. This is a nice fallback. Nice catch FR!